bitnami/spark/templates/statefulset-worker.yaml

```diff
@@ -182,11 +182,11 @@ spec:
           volumeMounts:
             {{- if .Values.worker.existingConfigmap }}
             - name: config
-              mountPath: '/bitnami/spark/conf/'
+              mountPath: '/opt/bitnami/spark/conf/'
```

Comment:
Could you please keep this one as `/bitnami/spark/conf/`? This path corresponds to `SPARK_MOUNTED_CONFDIR`, and files from `/bitnami/spark/conf/` will be copied to `/opt/bitnami/spark/conf/`  at runtime.

Reply:
Yup, let me change that.